stl/inc/vector

```diff
@@ -2124,6 +2130,14 @@ public:
         return this->_Myptr == _Right._Myptr && this->_Myoff == _Right._Myoff;
     }
 
+#if _HAS_CXX20
+    _NODISCARD strong_ordering operator<=>(const _Vb_const_iterator& _Right) const noexcept {
+        if (const auto _CmpResult = this->_Myptr <=> _Right._Myptr; _CmpResult != 0) {

```

Comment:
Note for other reviewers: `_Myptr` is a raw pointer, not a fancy pointer, so we can use `<=>` directly.